rust/core-lib/src/syntax.rs

```diff
@@ -41,7 +41,7 @@ pub struct LanguageDefinition {
 /// A repository of all loaded `LanguageDefinition`s.
 #[derive(Debug, Default)]
 pub struct Languages {
-    named: HashMap<LanguageId, Arc<LanguageDefinition>>,
+    pub named: HashMap<LanguageId, Arc<LanguageDefinition>>,
```

Comment:
this is unnecessary; `Languages` has an `iter` fn, which returns an iterator over the `LanguageDefinition` objects.